### PR TITLE
GEN-ART review from Russ

### LINKED ISSUE #117: Address comments Gen-ART review
See comments https://mailarchive.ietf.org/arch/msg/anima/87S9oAHrhRxMI03lRiutpClQRVc/

### PR DESCRIPTION
**Pull request by mcr**
changes as suggested by Russ, close #117

### CODE REVIEW

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -358,14 +363,15 @@ For instance, for a domain with a two-level certification authority (see {{fig-t
 ~~~~
 {: #fig-twoca title='Two-Level CA PKI'}
 
-When the Registrar is using a COSE-signed constrained voucher request towards MASA, instead of a regular CMS-signed voucher request, the COSE_Sign1 object contains a protected and an unprotected header, and according to {{I-D.ietf-cose-x509}},
-would carry all the certificates of the chain in an "x5bag" or "x5chain" attribute placed in the unprotected header.
+When the Registrar is using a COSE-signed constrained voucher request towards MASA, instead of a regular CMS-signed voucher request, the COSE_Sign1 object contains a protected and an unprotected header.
+The Registrar MUST place all all the certificates for the chain in an "x5bag" attribute in the unprotected header {{I-D.ietf-cose-x509}}.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:367]: all all -> all

File: pinning-options.txt [hunk 2]
```diff
@@ -1,14 +1,14 @@
- .---------------.
- | domain CA (1) |
- '---------------'
+ .------------.
+ | pub-CA (1) |
```
Comment (EskoDijk) [inline pinning-options.txt:2]: Why is domain CA renamed to pub-CA? Most typical domain CAs are not public i.e. not part of PKIX. Rather, they use a private root CA certificate typically.

**Review by EskoDijk — APPROVED** [review]
Ok! Only one comment/question on the pub-CA in the figure.

File: constrained-voucher.mkd (hunk 1 above)
Comment (mcr) [inline constrained-voucher.mkd:367]:
```suggestion
The Registrar MUST place all the certificates for the chain in an "x5bag" attribute in the unprotected header {{I-D.ietf-cose-x509}}.
```

**Pull request merged**